packages/api/package.json

```diff
@@ -10,27 +10,37 @@
   "types": "./dist/index.d.ts",
   "license": "MIT",
   "dependencies": {
+    "@envelop/apollo-server-errors": "1.0.2",
+    "@envelop/core": "1.0.3",
+    "@envelop/depth-limit": "1.0.0",
+    "@envelop/disable-introspection": "1.0.0",
+    "@envelop/filter-operation-type": "1.0.0",
+    "@envelop/parser-cache": "1.0.0",
+    "@envelop/validation-cache": "1.0.0",
     "@graphql-tools/merge": "7.0.0",
+    "@graphql-tools/schema": "8.0.3",
+    "@graphql-tools/utils": "8.0.2",
     "@prisma/client": "2.29.1",
     "@types/pino": "6.3.11",
     "apollo-server-lambda": "2.25.2",
```

Comment:
Note that we need to import `apollo-server-lambda` temporarily due to the use of Apollo Server errors and mimicking their behaviors in the error handling plugins for the GraphQL handler.

The `apollo-server-lambda` import will be removed once error handling on the graphql and web-side formError handling is refactored.